### ISSUE TITLE
It seems that LR configs are not effective.

### ISSUE THREAD

**Issue by ZhangGongjie** [2U1/Qwen-VL-Series-Finetune#81]
It seems that in trainer.py, self.optimizer is None is always False. Therefore learning rate setups are not used by the model.

**Comment by 2U1** [comment]
Thanks for the issue! I'll take a look.

**Comment by 2U1** [comment]
Maybe the optimizer config in the deepspeed config part overrides the optimizer config part.
I've erased the part from the deepspeed config file.
